src/garage/tf/policies/categorical_gru_policy2.py

```diff
@@ -0,0 +1,301 @@
+"""CategoricalGRUPolicy with model."""
+import akro
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.models import CategoricalGRUModel
+from garage.tf.policies import StochasticPolicy2
+
+
+class CategoricalGRUPolicy2(StochasticPolicy2):
+    """CategoricalGRUPolicy.
```

Comment:
you don't need to use the name as the summary, you can just summarize the class.

Reply:
Is this a good example?
`A categorical policy that contains a GRU`

Reply:
similar to that, yeah. 

how about things like `A policy represented by a Categorical distribution which is parameterized by a GRU.`

Reply:
That would be too long to fit in one line and if it’s a multi line summary, flake8 will complain...